src/main/java/com/dreammaster/bartworksHandler/BacteriaRegistry.java

```diff
@@ -286,7 +286,7 @@ private void runBWRecipes(){
         addBacterialVatRecipe(
                 new ItemStack[]{AntimonyTrioxide.getDust(16), Osmium.getDust(16)},
                 CultureSet.get("CombinedBac"),
-                new FluidStack[]{Oil.getFluid(1000)},
+                new FluidStack[]{Oil.getFluid(1)},
```

Comment:
this is change we need to discuss. You cahnage from 1000 to 1.
And this change not belongs to the xsmd's so please seperate it

Reply:
ok yea use 1000 is very much. I would use 10 as values but yea